views/actions/packages.js

```diff
@@ -115,6 +154,12 @@ function createAddPackageRequest(carrier, trackingNumber) {
   };
 }
 
+function sortHistoryByDate(history) {
+  return history.sort((a, b) => {
+    return dateComparator(a, b);
+  });
```

Comment:
How come we cannot sort this in the route when grabbing the model? It is much faster if we allow postgres to sort for us.